scripts/azure-ci-test.sh

```diff
@@ -24,12 +24,16 @@ SUFFIX=$(openssl rand -hex 2)
 export GROUP_NAME="${GROUP_NAME:-ratify-e2e-${SUFFIX}}"
 export ACR_NAME="${ACR_NAME:-ratifyacr${SUFFIX}}"
 export AKS_NAME="${AKS_NAME:-ratify-aks-${SUFFIX}}"
+# export KEYVAULT_NAME="${KEYVAULT_NAME:-ratify-akv-${SUFFIX}}"
+export KEYVAULT_NAME="ratify-e2e-kv"
+export USER_ASSIGNED_IDENTITY_NAME="${USER_ASSIGNED_IDENTITY_NAME:-ratify-e2e-identity-${SUFFIX}}"
 export LOCATION="eastus"
 export KUBERNETES_VERSION=${1:-1.24.6}
+export RATIFY_NAMESPACE=${4:-default}
```

Comment:
is the default namespace `default` or `gatekeeper-system`?

Reply:
thanks for catching! It's outdated.